res/TensorFlowLiteRecipes/Net_Conv_Mul_004/test.recipe

```diff
@@ -0,0 +1,70 @@
+# test for Conv(pad=valid, krnl(8, 3, 2, 4), stride=(2, 1))->Mul_with_constant
```

Comment:
3 and 4 has same comment, what are the differences?

Reply:
`Mul_constant ` in test 4 is rank-0 constant. It's done to make sure rank-0 constants are still passable.
I will change comment to `...->Mul_with_rank-0_constant`.